app/src/main/java/com/xmartlabs/scasas/geoquiz/module/Question.java

```diff
@@ -0,0 +1,46 @@
+package com.xmartlabs.scasas.geoquiz.module;
+
+/**
+ * Created by scasas on 1/25/17.
+ */
+
+public class Question {
+  private int textResId;
+  private boolean answerTrue;
+
+  public int getTextResId() {
+    return textResId;
+  }
+
+  public boolean isAnswerTrue() {
+    return answerTrue;
+  }
+
+  public static final class Builder {
+    private int textResId;
+    private boolean answerTrue;
+
+    public Builder() {
+    }
+
+    public Builder textResId(int val) {
```

Comment:
Prefer to use a more descriptive name for variables whenever possible. I'd use textResId here